.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "src/modules/mavlink/mavlink"]
 	path = src/modules/mavlink/mavlink
-	url = https://github.com/mavlink/mavlink.git
+	url = https://github.com/Auterion/mavlink.git
```

Comment:
I assume this will have to be changed to upstream mavlink, once it is merged in upstream mavliink, right? (Before we merge here)

Reply:
Exactly, this PR will stay in draft because of this. I need to update this as soon as it is merged on Mavlink